gamma/2.1/Dockerfile

```diff
@@ -0,0 +1,103 @@
+# base image
+FROM ubuntu:bionic as app
+
+# for easy upgrade later. ARG variables only persist during image build time
+ARG GAMMA_VER="2.1"
+ARG HMMR_VER="3.3"
+ARG AMR_VER="3.10.18"
+
+# metadata
+LABEL base.image="ubuntu:bionic"
+LABEL dockerfile.version="1"
+LABEL software="GAMMA"
+LABEL software.version="v2.1"
+LABEL description="Gene Allele Mutation Microbial Assessment"
+LABEL website="https://github.com/rastanton/GAMMA/"
+LABEL license="Apache 2.0"
+LABEL maintainer="Jill Hagey"
+LABEL maintainer.email="jvhagey@gmail.com"
+
+# install dependencies
+RUN apt-get update && apt-get -y --no-install-recommends install \
+    pigz \
+    rsync \
+    unzip \
+    tar \
+    python3.7 \
+    python3-pip \
+    gzip \
+    wget \
+    git \
+    curl \
+    make \
+    g++ \
+    ncbi-blast+ \
+    hmmer && \

```

Comment:
You can remove `hmmer` from the `apt-get install` step since you are installing it manually down below. Ubuntu:bionic has hmmer 3.1, so I think it's best to remove it from this step.

https://packages.ubuntu.com/bionic/hmmer